src/code/code_80140CE0.c

```diff
@@ -0,0 +1,86 @@
+#include "global.h"
+
+extern Gfx D_0E0001C8[];
+
+Gfx D_801C5DD0[] = {
+    gsDPSetOtherMode(G_AD_PATTERN | G_CD_MAGICSQ | G_CK_NONE | G_TC_CONV | G_TF_POINT | G_TT_NONE | G_TL_TILE |
+                         G_TD_CLAMP | G_TP_NONE | G_CYC_1CYCLE | G_PM_NPRIMITIVE,
+                     G_AC_NONE | G_ZS_PRIM | G_RM_VISCVG | G_RM_VISCVG2),
+    gsSPBranchList(0x0E0002E0),
```

Comment:
```suggestion
    gsSPBranchList(D_0E000000.fillRect),
```
All uses of `0x0E0002E0` in this file should be this